pkgs/servers/db-rest/default.nix

```
@@ -0,0 +1,49 @@
+{ lib
+, buildNpmPackage
+, fetchFromGitHub
+, nodejs_18
+, nix-update-script
+}:
+let
+  buildNpmPackage' = buildNpmPackage.override { nodejs = nodejs_18; };
+in buildNpmPackage' rec {
+  pname = "db-rest";
+  version = "6.0.2";
+
+  src = fetchFromGitHub {
+    owner = "derhuerst";
+    repo = pname;
+    rev = version;
+    hash = "sha256-xgWMLGk4ksexOg0+KIPEKaW2d16QWdCElAivj/nle0k=";
+  };
+
+  npmDepsHash = "sha256-ZWD1FmGJX+GUSa1h/dDnfXVIWKR6Z92nN5YP3bbGwcU=";
+  dontNpmBuild = true;
+
+  buildPhase = ''
+    runHook preBuild
+    REDIS_URL="" ${nodejs_18}/bin/node ./build/index.js
```

```suggestion
    REDIS_URL="" node ./build/index.js
```

Quoting like that will use the wrong `node` (e.g. in cross scenarios)

(To be clear though the usage in `installPhase` is fine and will use the correct `node` since string coercion chooses host `node` rather by build `node`)